src/main/resources/assets/minestuck/models/block/lunchtop_bsod.json

```diff
@@ -1,2415 +0,0 @@
-				"south": {
-					"uv": [0, 0, 15.5, 15.5],
-					"texture": "#2"
-				},
-				"west": {"uv": [0, 0, 15.5, 15.5], "texture": "#2"},
-				"up": {"uv": [0, 0, 15.5, 15.5], "texture": "#2"},
-				"down": {"uv": [0, 0, 15.5, 15.5], "texture": "#2"}
-			},
-			"type": "cube"
-		},
-		{
-			"name": "cube",
-			"from": [
-				5.5249999999999995,
-				4.500000000000001,
-				3.125
-			],
-			"to": [6.0249999999999995, 4.7, 3.625],
-			"faces": {
-				"north": {
-					"uv": [0, 0, 15.5, 15.5],
-					"texture": "#2"
-				},
-				"east": {"uv": [0, 0, 15.5, 15.5], "texture": "#2"},
-				"south": {
-					"uv": [0, 0, 15.5, 15.5],
-					"texture": "#2"
-				},
-				"west": {"uv": [0, 0, 15.5, 15.5], "texture": "#2"},
-				"up": {"uv": [0, 0, 15.5, 15.5], "texture": "#2"},
-				"down": {"uv": [0, 0, 15.5, 15.5], "texture": "#2"}
-			},
-			"type": "cube"
-		},
-		{
-			"name": "cube",
-			"from": [0.25, 7, 12.45],
-			"to": [15.75, 16, 13],
-			"faces": {
-				"north": {
-					"uv": [0, 2.1875, 16, 12.05],
-					"texture": "#3"
-				},
-				"east": {
-					"uv": [0.125, 2.1875, 0.625, 2.7375],
-					"texture": "#3"
-				},
-				"south": {
-					"uv": [16, 2.1875, 0, 12.05],
-					"texture": "#3"
-				},
-				"west": {
-					"uv": [0.125, 2.1875, 0.625, 2.7375],
-					"texture": "#3"
-				},
-				"up": {
-					"uv": [0.125, 2.1875, 0.625, 2.7375],
-					"texture": "#3"
-				},
-				"down": {
-					"uv": [0.125, 2.1875, 0.625, 2.7375],
-					"texture": "#3"
-				}
-			},
-			"type": "cube"
-		}
-	],
-	"groups": [
-		0,
-		1,
-		2,
-		3,
-		4,
-		5,
-		6,
-		7,
-		8,
-		9,
-		10,
-		11,
-		12,
-		13,
-		14,
-		15,
-		16,
-		17,
-		18,
-		19,
-		20,
-		21,
-		{
-			"name": "group",
-			"isOpen": true,
-			"display": {"visibility": true, "autouv": true},
-			"children": [
-				22,
-				23,
-				24,
-				25,
-				26,
-				27,
-				28,
-				29,
-				30,
-				31,
-				32,
-				33,
-				34,
-				35,
-				36,
-				37,
-				38,
-				39,
-				40,
-				41,
-				42,
-				43,
-				44,
-				45,
-				46,
-				47,
-				48,
-				49,
-				50,
-				51,
-				52,
-				53,
-				54,
-				55,
-				56,
-				57,
-				58,
-				59,
-				60,
-				61,
-				62,
-				63,
-				64,
-				65,
-				66,
-				67,
-				68,
-				69,
-				70,
-				71,
-				72,
-				73,
-				74,
-				75,
-				76,
-				77,
-				78,
-				79,
-				80,
-				81,
-				82,
-				83,
-				84,
-				85,
-				86,
-				87,
-				88,
-				89,
-				90,
-				91,
-				92,
-				93,
-				94,
-				95,
-				96,
-				97,
-				98,
-				99
-			]
-		}
-	],
-	"display": {
-		"thirdperson_righthand": {
-			"scale": [0.5, 0.5, 0.5],
-			"rotation": [57, 0, 0],
-			"translation": [0, 4, 2.5]
-		},
-		"thirdperson_lefthand": {
-			"scale": [0.5, 0.5, 0.5],
-			"rotation": [57, 0, 0],
-			"translation": [0, 4, 2.5]
-		},
-		"ground": {
-			"scale": [1, 1, 1],
-			"translation": [0, 4, 0]
-		},
-		"gui": {
-			"scale": [0.625, 0.625, 0.625],
-			"rotation": [30, 225, 0],
-			"translation": [0, 0, 0]
-		}
-	}
```

Comment:
In the GUI it appears flat, facing directly towards the screen instead of in an isometric view as most blocks are (alternatively we could use an item sprite if you wished). It also floats and appears large when viewed in third person. Blockbench has a feature where you can apply a "block" preset to display which should fix this